### ISSUE TITLE
How i can add navigation

### ISSUE THREAD

**Issue by monikagupta78** [explooosion/Agm-Direction#33]
I used this library to show route, next i am looking for navigation. can you pls guide how we can add navigation for that route?

**Comment by explooosion** [comment]
Hi ! @monikagupta78 

I'm not sure I understand what you mean.
You want to display a directions [panel](https://developers.google.com/maps/documentation/javascript/examples/directions-panel?hl=en)?🤔

Here is an example of panel:

```html
<agm-direction ... [panel]="myPanel"></agm-direction>
<div #myPanel></div>
```

Check out more example:

- Agm-Direction-Docs: [Direction](https://robby570.tw/Agm-Direction-Docs/source/featured/direction.html) (Update Panel With New Query)

Hope to help you~

**Comment by monikagupta78** [comment]
No, I am looking to add Navigation, like we have in android phone in google maps, for user to navigation from point A to B with geo tracking on.

**Comment by explooosion** [comment]
Sorry, I can't understand clearly.
Could you provide some use case or images about this issue?

**Comment by monikagupta78** [comment]
My usecase is to draw a route with multiple stops and provide real time navigation like Ola/Uber.
I am trying to understand if I can add real time navigation after drawing the route using this direction library? 
 one option could be to generate a URL, that can take the user to google Maps navigation App.

Need help here to construct a URL with multiple waypoints using this module? like it is available in Google direction API using dir_action=navigate attribute?

https://developers.google.com/maps/documentation/urls/guide#directions-action 

example URL - https://www.google.com/maps/dir/?api=1&origin=Paris,France&destination=Cherbourg,France&travelmode=driving&waypoints=Versailles,France%7CChartres,France%7CLe+Mans,France%7CCaen,France 

Any inputs pls?
